### PR TITLE
replace TSX with ts-blank-space

### PR DESCRIPTION
**Pull request by turadg**
incidental

## Description
While creating a fast-usdc package, I ran into some trouble with the `tsx` loader for Ava.

So I tried https://github.com/bloomberg/ts-blank-space instead and it worked perfectly.

This replaces all uses of the tsx loader so the root Yarn project no longer has tsx or its esbuild dependency.

The new loader is more strict about types so this also enables `verbatimModuleSyntax` in tsconfig to enable stricter imports while editing. The bulk of the changes are bringing imports into compliance.

### Security Considerations
Removes `esbuild` which installed many native binaries.

### Scaling Considerations

This does not affect application runtime. Here's the difference in testing `@agoric/orchestration` with `hyperfine --runs 5 "yarn test"`:

```
# master
  Time (mean ± σ):     53.074 s ±  1.746 s    [User: 268.841 s, System: 77.395 s]
  Range (min … max):   50.596 s … 55.502 s    5 runs
# PR
  Time (mean ± σ):     46.673 s ±  3.855 s    [User: 234.273 s, System: 49.280 s]
  Range (min … max):   44.074 s … 53.320 s    5 runs
```

So it's faster too!

### Documentation Considerations
not end user facing

### Testing Considerations
CI

### Upgrade Considerations
n/a

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
I looked up "ts blank space" to find out what this was about, and had to watch a music video.

**Comment by michaelfig** [comment]
I'd appreciate a note in "Scaling Considerations" to characterise the relative performance cost or benefit of this new approach, or at least "Unmeasured" if it was not/cannot be measured.

**Review by michaelfig — APPROVED** [review]
Looks great!  It's always nice to see the hatches battened down.

File: yarn.lock [hunk 1]
```diff
@@ -11590,6 +11346,13 @@ ts-api-utils@^1.3.0:
   resolved "https://registry.yarnpkg.com/ts-api-utils/-/ts-api-utils-1.3.0.tgz#4b490e27129f1e8e686b45cc4ab63714dc60eea1"
   integrity sha512-UQMIo7pb8WRomKR1/+MFVLTroIvDVtMX3K6OUir8ynLyzB8Jeriont2bTAtmNPa1ekAgN7YPDyf6V+ygrdU+eQ==
 
+ts-blank-space@^0.4.1:
+  version "0.4.1"
+  resolved "https://registry.yarnpkg.com/ts-blank-space/-/ts-blank-space-0.4.1.tgz#3ddc6c7c4009337ffb1811071a8fc55d8076b619"
+  integrity sha512-i0JvTvFaZcXCj+tcs67G0K3zXgUr1ebGvhmuB3ov0ByFrbuvTOgUwtFEOP8dN+B+ou0wkhosJ8Q4RPfi8mHFQQ==
+  dependencies:
+    typescript "5.1.6 - 5.6.x"
```
Comment (mhofman) [inline yarn.lock:11354]: This might become interesting when we try to upgrade tsc and blank-space hasn't caught up.

**Review by mhofman — APPROVED** [review]
🎉

**Pull request merged**